site/src/sphinx/server-grpc.rst

```diff
@@ -0,0 +1,88 @@
+.. _GRPC: http://www.grpc.io/
+.. _GrpcService: apidocs/index.html?com/linecorp/armeria/server/grpc/GrpcService.html
+.. _GrpcServiceBuilder: apidocs/index.html?com/linecorp/armeria/server/grpc/GrpcServiceBuilder.html
+.. _ServerBuilder: apidocs/index.html?com/linecorp/armeria/server/ServerBuilder.html
+
+.. _server-grpc:
+
+Running a GRPC service
+======================
+
+Let's assume we have the following GRPC_ service definition:
+
+.. code-block:: protobuf
+
+    syntax = "proto3";
+
+    package com.example.grpc.hello;
```

Comment:
Nit: The proto convention is to not use a reverse domain name for the proto/c++ package. So it'd be more conventional for this to be just ```grpc.hello```.